XCUITests/FxScreenGraph.swift

```diff
@@ -332,4 +333,36 @@ extension Navigator {
             self.goto(TabTray)
         }
     }
+
+    func selectOptionFromBrowserTabMenu(view: selectMenuOption) {
```

Comment:
Thanks for the tips! and for the names suggested, changes done.  Hope the final ones are good to you:
browserPerformAction and for the enum, the same, BrowserPerformAction.